default.yaml

```diff
@@ -11,11 +11,14 @@ buildVersion:
     minBuildNumber: 182
     lastBuildNumber: 269
 
+rewards:
+  whitelistedCountries: [SV]
```

Comment:
can we agree on the more descriptive `allow` / `deny` instead of the loaded `black` / `white`?